user_service/prisma/schema.prisma

```diff
@@ -21,4 +21,18 @@ model User {
   username        String   @db.VarChar(255) @unique
   hashedPassword  String   
   role            Role     @default(USER)
+}
+
+
+model AnsweredQuestion {
+  id         Int        @id @default(autoincrement())
+  userId     Int
+  user       User     @relation(fields: [userId], references: [id])
+  questionTitle String
+  questionId Int
+  complexity Complexity
+  category   String[]
```

Comment:
![image](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g10/assets/72195240/e5ff7a22-0994-41bb-923f-4519614a3078)

What is type `Complexity`?

Reply:
edited in latest pr